crud/__init__.py

```diff
@@ -0,0 +1,24 @@
+from .link_crud import (
+    add_clicks_date,
+    campaign_add,
+    delete_link_db,
+    get_campaign_list,
+    get_clicks_date,
+    get_default_values,
```

Comment:
Add import to init file is very discussable thing, my own think - it is not a good idea